webapp/src/utils.ts

```diff
@@ -162,12 +156,6 @@ export function stateSortProfiles(profiles: UserProfile[], statuses: { [key: str
             };
         }
 
-        if (stateA.unmuted && !stateB.unmuted) {
-            return -1;
-        } else if (stateB.unmuted && !stateA.unmuted) {
-            return 1;
-        }
-
         if (stateA.raised_hand && !stateB.raised_hand) {
```

Comment:
I suppose it's far less likely or useful but should we have this take precedence over the presenter state as well?

Reply:
I think my reasoning is purely to keep the queue consistency at all costs but usability wise yeah, I don't think it should ever be a problem. Let's merge this as is and we can revisit anytime if needed.